deployment/aws-python-storage-stack-automation/storage_stack.yaml

```diff
@@ -0,0 +1,319 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: "Automatically deploy File Storage Security - Storage stack to every new S3 bucket created and register it to the defined Scanner Stack."
+Metadata:
+  License: MIT-License
+  Version: 1.2
+Resources:
+  apiparameter:
+    Type: AWS::SSM::Parameter
+    Properties: 
+      Description: C1 WS API KEY
+      Type: String
+      Value: !Ref C1WSAPI
```

Comment:
Please use secret manager because ssm SecureString iis not supported. You can reference this plugin.
https://github.com/trendmicro/cloudone-filestorage-plugins/blob/master/post-scan-actions/aws-python-conformity-custom-check/template.yml#L55

Reply:
Sorry about that. I changed it over to secrets manager. Thank you for the pointer.